src/sap/fhir/model/r4/FHIRUtils.js

```diff
@@ -717,6 +717,42 @@ sap.ui.define([
 		return sFullUrl;
 	};
 
+	/**
+	* Extracts resource IDs from the error description in FHIR OperationOutcome.
+	* @param {Array<object>} aOperationOutcomes - The array of FHIR OperationOutcome objects.
+	* @returns {Array<string>} - An array containing the extracted resource IDs.
+	* @private
+	* @since 2.3.8
+	*/
+	FHIRUtils.getsIdFromOperationOutcome = function(aOperationOutcomes){
+		var sIds = [];
+		for (var key in aOperationOutcomes) {
+			if (aOperationOutcomes.hasOwnProperty(key)) {
+				var operationOutcome = aOperationOutcomes[key];
+			    var text = operationOutcome._aIssue[0].details.text;
+				var sId = text.match(/[0-9a-fA-F]{8}(?:-[0-9a-fA-F]{4}){3}-[0-9a-fA-F]{12}/);
```

Comment:
what if the issue is empty ? and also the issue doesnt have the resource id ? then sID[o] will throw exception ?